packages/charts/src/components/common/components/Tooltip/Tooltip.tsx

```diff
@@ -91,8 +93,11 @@ export const Tooltip = ({
         tooltipContext.tooltipData?.datumByKey,
         childSumLabel,
     );
+    const entriesWithFormattedSeries = valueContextBySeries
```

Comment:
You could move this into getTooltipEntries and potentially even memoize that function. That way you have one function that returns your formatted tooltip content

Reply:
Moved, didn't do the memo though, no reason to do it unless it's slow as it also has a overhead cost.

Reply:
nothing is slow on a powerful enough cpu 😛 `.reverse` is efficient because it is performed in-place but i would suggest the memory cost of `useMemo` are more than surpassed by the memory spent spreading that array every time the component re-renders. Then it's off waiting for the garbage collector, potentially piles of lonely arrays with nothing pointing to them